### PR TITLE
Mike/navbar footer refactor pt.2

### PR DESCRIPTION
**Pull request by ghost**
Nav and footer now accommodate 280px viewports. Combined with index page commit, fixes glitchy behavior on Galaxy Fold

### CODE REVIEW

File: src/lib/components/sections/navbar.svelte [hunk 1]
```diff
@@ -32,7 +32,7 @@
     <div class="logo-container">
       <a href="/" class="logo">
         <img src="assets/badges/general.svg" class="badge" alt="acmCSUF logo" />
-        <h3>CSUF</h3>
+        <span class="brand-header size-md">at <b>CSUF</b></span>
```
Comment (EthanThatOneKid) [inline src/lib/components/sections/navbar.svelte:35]: Does `<b>CSUF</b>` accomplish the same thing as our usual `<span class="brand-em">CSUF</span>`?

File: src/lib/components/sections/navbar.svelte [hunk 2]
```diff
@@ -93,87 +93,78 @@
...
-      display: none;
-      z-index: 10;
-    }
+  .toggle,
+  .menu {
+    display: none;
+    z-index: 10;
   }
 
   @media screen and (max-width: 900px) {
```
Comment (EthanThatOneKid) [inline src/lib/components/sections/navbar.svelte:160]: This codebase uses whichever breakpoint value seems to be convenient in their respective components/pages. Should we use a constant breakpoints for mobile, tablet, desktop, and ultrawide in a future update?

**Review by EthanThatOneKid — APPROVED** [review]
Seems to be a nice collection of design tweaks, looking forward to merging them.

I added some food-for-thought in my review, however.

**Comment by EthanThatOneKid** [comment]
> [12:57 PM] moike: Okok go ahead and close 426, 415, 414, 413, and 406

File: src/lib/components/sections/navbar.svelte (hunk 1 above)
Comment (ghost) [reply src/lib/components/sections/navbar.svelte:35]: Yes

File: src/lib/components/sections/navbar.svelte (hunk 2 above)
Comment (ghost) [reply src/lib/components/sections/navbar.svelte:160]: I plan to make the standard breakpoint 900px for all

**Pull request closed without merging**